Source/Core/DiscIO/Volume.h

```diff
@@ -35,8 +35,8 @@ struct Partition final
   constexpr bool operator!=(const Partition& other) const { return !(*this == other); }
   constexpr bool operator<(const Partition& other) const { return offset < other.offset; }
   constexpr bool operator>(const Partition& other) const { return other < *this; }
-  constexpr bool operator<=(const Partition& other) const { return !(*this < other); }
-  constexpr bool operator>=(const Partition& other) const { return !(*this > other); }
+  constexpr bool operator<=(const Partition& other) const { return *this >= other; }
+  constexpr bool operator>=(const Partition& other) const { return *this <= other; }
```

Comment:
Does this even work? We only implemented `operator<` technically, with `operator>` and those two calling it inversely.